docs/002-projects.apib

```diff
@@ -135,6 +135,41 @@ Deletes a device and de-provisions it in our datacenter.
 
 + Response 204
 
+## Events [/projects/{projectId}/events]
```

Comment:
This should be `{id}`, not `{projectId}`. Same for the entry under `+ Parameters` below.